### PR TITLE
Improvements testing package

### PR DESCRIPTION
**Pull request by marioidival**
~~Travis build of Go 1.6 it's broken, because I'm using `t.Run` feature of `testing` package.~~

### CODE REVIEW

File: adapters/postgres/postgres_test.go [hunk 1]
```diff
@@ -10,781 +10,782 @@ import (
...
+	}{
+		{"$eq", "="},
+		{"$ne", "!="},
+		{"$gt", ">"},
+		{"$gte", ">="},
+		{"$lt", "<"},
+		{"$lte", "<="},
+		{"$in", "IN"},
+		{"$nin", "NOT IN"},
+	}
```
Comment (felipeweb) [inline adapters/postgres/postgres_test.go]: add tests about $null and $notnull operators

**Comment by avelino** [comment]
thank you @marioidival
